git.go

```diff
@@ -385,7 +385,7 @@ func (s *GitRepo) ExportDir(dir string) error {
 		return NewLocalError("Unable to export source", err, string(out))
 	}
 	// and now, the horror of submodules
-	out, err = s.RunFromDir("git", "submodule", "foreach", "--recursive", "'git checkout-index -f -a --prefix=\""+filepath.Join(dir, "$path")+"\"'")
+	out, err = s.RunFromDir("git", "submodule", "foreach", "--recursive", "git checkout-index -f -a --prefix=\""+filepath.Join(dir, "$path")+"/\"")
```

Comment:
Whoops, yes, it is, I just forgot to mention it in the message / I probably should have included a test.

I found via experimentation that without it, the submodule's files end up prefixed and in the parent dir.